src/styles/layout.scss

```diff
@@ -349,7 +349,7 @@ nav,
 .nav-item {
   display: flex;
   flex: 1 0 auto;
-  justify-content: center;
+  justify-content: space-around;
```

Comment:
I'm not sure this is doing anything here.

Reply:
Yeah - maybe not. I thought that too.

Reply:
Do you think I should delete? I was too :fearful:

Reply:
If you want to! It seems there's only one flex child so the spacing isn't going to change. Perhaps the icon and text used to be the direct children of the nav-item.